tools/ci/security_audit.sh

```diff
@@ -42,7 +43,7 @@ scan_code()
         if should_ignore ${file}; then continue; fi
         if [ ! -f ${file} ];then continue; fi
         LOG_INFO "check file ${file}"
-        python ${scan_code_script} $file 
+        python ${scan_code_script} "$file"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=5426292)